src/Identity/Program.cs

```diff
@@ -22,22 +21,21 @@ public static IHostBuilder CreateHostBuilder(string[] args)
             {
                 webBuilder.UseStartup<Startup>();
                 webBuilder.ConfigureLogging((hostingContext, logging) =>
-                    logging.AddSerilog(hostingContext, e =>
+                    logging.AddSerilog(hostingContext, (e, globalSettings) =>
                     {
                         var context = e.Properties["SourceContext"].ToString();
-                        if (context.Contains(typeof(IpRateLimitMiddleware).FullName) &&
-                            e.Level == LogEventLevel.Information)
+                        if (context.Contains(typeof(IpRateLimitMiddleware).FullName))
                         {
-                            return true;
+                            return e.Level >= globalSettings.MinLogLevel.IdentitySettings.IpRateLimit;
                         }
 
                         if (context.Contains("IdentityServer4.Validation.TokenValidator") ||
                             context.Contains("IdentityServer4.Validation.TokenRequestValidator"))
                         {
-                            return e.Level > LogEventLevel.Error;
+                            return e.Level >= globalSettings.MinLogLevel.IdentitySettings.IdentityToken;
```

Comment:
Also changed from `Error` to `Fatal`.

Reply:
These are actually consistent, original was `>` not `>=`, but I wanted consistency in **Min**LoggingLevel

Reply:
Ah good point